The Basics/ex4.py

```diff
@@ -0,0 +1,27 @@
+# Challenge 004 
+
+"""
+ask the user to enter two numbers then display 
+the answer as 'The total is [answer]'
+"""
+
+# my answer
+num1 = input("Please enter a number ")
+num2 = input("Please enter another number ")
+total_num = (int(num1) + int(num2))
```

Comment:
You also don't need to contain these calculations inside parenthesis: `int(num1) + int(num2)` will work the same